### PR TITLE
Register Subdomain: atomu

### PR DESCRIPTION
**Pull request by aatomu**
Added the domain: atomu

### CODE REVIEW

File: domains/atomu.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "atomu21263",
+        "email": "atom.aizu@gmail.com"
+      },
+
+      "record": {
+        "A": ["106.73.4.128"]
```
Comment (WilliamDavidHarrison) [inline domains/atomu.json:9]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/atomu.json (hunk 1 above)
Comment (atomu21263) [reply domains/atomu.json:9]: use port :8816
(my HGW doesnt open 80,433

Comment (WilliamDavidHarrison) [reply domains/atomu.json:9]: @atomu21263 Sorry, your website needs to be in English.

Comment (atomu21263) [reply domains/atomu.json:9]: hmm other my web service :29300?
source: https://github.com/atomu21263/paint

Comment (WilliamDavidHarrison) [reply domains/atomu.json:9]: @atomu21263 That will work, but you need to add a main page.

Comment (atomu21263) [reply domains/atomu.json:9]: main page URL is ...4.128:430/?

Comment (WilliamDavidHarrison) [reply domains/atomu.json:9]: > main page URL is "...4.128:430/"?

Oh. It needs to be on the same IP, I believe.

Comment (atomu21263) [reply domains/atomu.json:9]: what? (i'm not good english,sorry.
my home Router(?) is doesnt open 80,433 and other Well Known Ports...

Comment (WilliamDavidHarrison) [reply domains/atomu.json:9]: > what? (i'm not good english,sorry.
> my home Router(?) is doesnt open 80,433 and other Well Known Ports...

@atomu21263 I would highly recommend, **not** using your router for your ip.

Comment (atomu21263) [reply domains/atomu.json:9]: hmm how to do? or what search keyword?

Comment (WilliamDavidHarrison) [reply domains/atomu.json:9]: You should change your IP immediately since you'll probably get DDOSed

Comment (atomu21263) [reply domains/atomu.json:9]: I can't change it because of my IPv6 contract, so can you delete this Pull request?

Comment (WilliamDavidHarrison) [reply domains/atomu.json:9]: You cannot delete PRs. Only close them, so I recommend closing this PR and hope that no one DDOSes you.

Comment (atomu21263) [reply domains/atomu.json:9]: ok, thank you support.
pls close PRs after check this message

Comment (WilliamDavidHarrison) [reply domains/atomu.json:9]: I can't close PR's. You have to. Use the close PR button at the bottom of the page.

Comment (atomu21263) [reply domains/atomu.json:9]: ok, thx.

**Pull request closed without merging**